web/src/apps/main/redux/actions/documentation-page/index.ts

```diff
@@ -61,7 +61,7 @@ const fetchCurrentDocumentContributors = (): ThunkResult<
   if (!currentDocument || Array.isArray(currentDocument.contributors)) return;
 
   const response = await Axios.get<GithubUser[]>(
-    apiURL + `/contributors?documentSlug=${currentDocument.slug}`,
+    apiURL + `/v2/contributors?documentSlug=${currentDocument.slug}`,
```

Comment:
Same here, but this time it's documentation:

```suggestion
    apiURL + `/v2/contributors?path=documentation/${currentDocument.slug}`,
```